spec/features/projects/template_spec.rb

```diff
@@ -126,7 +126,10 @@
       expect(page).to have_content I18n.t(:label_copy_project)
       expect(page).to have_content I18n.t('js.job_status.generic_messages.in_queue')
 
-      # Email notification should be sent
+      # Perform the background job
+      perform_enqueued_jobs
+
+      # Perform the enqueued mailer job
       perform_enqueued_jobs
```

Comment:
Yes, the call is required twice. The first runs the project job, which now enqueues a mailer job instead of sending it out synchronously. The second call then processes the mailer job. I changed the code to match another use in this pattern which uses `2.times { }`. I think this makes it more clear